modules/accumulo/src/main/java/io/fluo/recipes/accumulo/export/DifferenceExport.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2016 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.recipes.accumulo.export;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import com.google.common.base.Preconditions;
+import io.fluo.api.data.Bytes;
+import io.fluo.api.data.Column;
+import io.fluo.api.data.RowColumn;
+import org.apache.accumulo.core.data.Mutation;
+
+/**
+ * Implemented by users to export data to Accumulo by comparing the the differences between a
+ * RowColumn/Bytes map that is generated for old and new data and represents how the data should
+ * exist in Accumulo.
+ * 
```

Comment:
It might be nice to list the different cases and whats done. Like

 * When old and new data generate same row col val, then nothing is done for that RCV.
 * When old and new data generate same row col with diff vals, then only an update mutation for the RCV is generated.
 * When old data generates a row col that new data does not, a delete mutation is generated.
 * When new data generated a row cold that old data does not, then an insert mutation is generated.

Could also mention that only one mutation is generated per row and seq is used as timestamp.